sticker-graph/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+setup(name='sticker-graph',
+      version='0.2.0',
+      description='Graph construction utilities for sticker',
+      url='https://github.com/danieldk/sticker',
+      author='Daniël de Kok',
+      author_email='me@danieldk.eu',
+      license='Apache2',
+      tests_require=[
+          'numpy',
+          'tensorflow == 1.13.1',
+          'toml',
+      ],
+      install_requires=[
+          'tensorflow == 1.13.1',
```

Comment:
But I want the CPU version ;), running a GPU version of Tensorflow to write the graph is a waste. I don't really like the SO solution, because then we have to maintain a list of different Tensorflow versions out there. E.g., there is also `intel-tensorflow` (which is probably compiled with Intel MKL-DNN).

At any rate, we should encourage that people use per-project virtual environments, which is especially easy since Python has included the `venv` module

Reply:
I'd still prefer not downloading an unnecessary dependency, but I guess it's not that important.

Reply:
Allow me to brag a little ;):

~~~bash
~ % nix path-info nixpkgs.python3Packages.tensorflow
/nix/store/fn697pz7j69zwjn703nmxcym3bdfhps8-python3.7-tensorflow-1.13.1
~ % nix path-info nixpkgs.python3Packages.tensorflowWithCuda
/nix/store/za3w1py46mfwf5r0g1fr7arrj6f07njb-python3.7-tensorflow-1.13.1
~~~

Shared between all my Tensorflow projects.